### PR TITLE
AppendTags improvements

### PR DESCRIPTION
**Pull request by invidian**
Moves from `util` package, which we shouldn't be using to just `pkg/platform` and adds tests for it.

Part of #290.

### CODE REVIEW

File: pkg/platform/platform.go [hunk 1]
```diff
@@ -61,3 +62,18 @@ func GetPlatform(name string) (Platform, error) {
 	}
 	return platform, nil
 }
+
+// AppendTags append lokoctl-version tag to given tags map.
+func AppendTags(tags *map[string]string) {
```
Comment (iaguis) [inline pkg/platform/platform.go:67]: Now that we're at it, can we change it to a better name?

For example: `AppendVersionTag` or something similar?

Comment (invidian) [reply pkg/platform/platform.go]: Done.

File: pkg/platform/platform_test.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	"testing"
+)
+
+func TestAppendVersionTagUninitializedMap(t *testing.T) {
+	var f map[string]string
+
+	AppendVersionTag(&f)
+
+	if len(f) == 0 {
+		t.Fatalf("version tag should be appended to uninitialized map")
```
Comment (iaguis) [inline pkg/platform/platform_test.go:13]: Nit, for consistency

```suggestion
		t.Fatalf("should append version tag to uninitialized map")
```

Comment (invidian) [reply pkg/platform/platform_test.go]: Thanks.

File: pkg/platform/platform.go [hunk 3]
```diff
@@ -61,3 +62,18 @@ func GetPlatform(name string) (Platform, error) {
 	}
 	return platform, nil
 }
+
+// AppendVersionTag append lokoctl-version tag to given tags map.
```
Comment (iaguis) [inline pkg/platform/platform.go:66]:
```suggestion
// AppendVersionTag appends the lokoctl-version tag to a given tags map.
```

File: pkg/platform/platform_test.go [hunk 4]
```diff
@@ -0,0 +1,33 @@
+package platform_test
```
Comment (iaguis) [inline pkg/platform/platform_test.go:1]: Can you add a copyright header?

**Pull request merged**